maykin_common/templates/maykin_common/api/index_base.html

```diff
@@ -0,0 +1,99 @@
+{% extends "master.html" %}
+{% load i18n static %}
+{% comment %}
+Base template for the landing page of an API project.
+
+It assumes that the default-project `master.html` root template exists in the project.
+Example usage:
+
+    {% extends "maykin_common/api/index_base.html" %}
+
+    {% block page_title %}My awesome API{% endblock %}
+
+    {% block body_en %}
+        <p>This API is awesome because ...</p>
+    {% endblock %}
+{% endcomment %}
+
+{% block view_class %}{{ block.super }} landing-page{% endblock %}
+
+{% block extra_css %}
+{{ block.super }}
+<link href="{% static 'maykin_common/css/api.css' %}" media="all" rel="stylesheet" />
+{% endblock %}
+
+{% block content %}
+
+    <header class="page-title">
+        <h1 class="page-title__title">
+            {% block page_title %}Override block page_title{% endblock page_title %}
+        </h1>
+        <p class="page-title__description">
+            {% block page_subtitle %}Override block page_subtitle{% endblock page_subtitle %}
+        </p>
+    </header>
+
+    <article class="page-content page-content--centered">
+        <nav class="tabs">
+            <ul class="tabs__list">
+                <li class="tabs__item tabs__item--active" data-id="nl" aria-role="tab">Nederlands</li>
+                <li class="tabs__item" data-id="en" aria-role="tab">English</li>
```

Comment:
the markup here is actually wrong, it should be `role` instead of `aria-rol` iirc :thinking: